astropy/timeseries/periodograms/lombscargle/core.py

```diff
@@ -399,15 +399,16 @@ def model(self, t, frequency):
 
         Parameters
         ----------
-        t : array_like or Quantity, length n_samples
-            times at which to compute the model
+        t : array-like or `~astropy.units.Quantity`.
+            Times at which to compute the model (should have length n_samples).
         frequency : float
             the frequency for the model
 
         Returns
         -------
-        y : np.ndarray, length n_samples
+        y : np.ndarray
             The model fit corresponding to the input times
+            (will have length n_samples).
```

Comment:
Should there be double backquote around `n_samples`?

Reply:
✅